pkg/http/api.go

```diff
@@ -100,6 +103,11 @@ func (a *api) APIEndpoints() []Endpoint {
 	return a.endpoints
 }
 
+// SetReadyStatus sets the ready status of dapr
+func (a *api) SetReadyStatus(status bool) {
```

Comment:
Since the status will only be marked as true (and never to false again) by runtime, can this method be changed to markStatusAsReady() without any arg which sets readyStatus to true.